packages/EtendoHookBinder/src/api/datasource/client.ts

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+import { API_BASE_URL } from "../constants";
+import { buildParams } from "./helpers";
+
+export const datasource = axios.create({
+  baseURL: API_BASE_URL,
+  // To Do: Implemenet JWT authentication for org.openbravo.service.datasource
+  withCredentials: true,
```

Comment:
@sjcuello I have some issues around this. 
I could not get the authorization working by adding the Authorization header using like:
```
'Authorization': `Bearer ${token}`
```
I kept receiving a 401 from the backend. Given this situation, I applied a workaround with `withCredentials: true` and manually login into etendo classic to get the cookies set.
The JWT authorization method works fine for SWS endpoints, but not for:
**/etendo/org.openbravo.client.kernel/OBUIAPP_MainLayout**